tutorials/dataframe/df105_WBosonAnalysis.py

```diff
@@ -191,7 +206,8 @@ def merge_histos(label):
 text.SetTextFont(42)
 text.DrawLatex(0.21 + 0.16, 0.86, "Open Data")
 text.SetTextSize(0.04)
-text.DrawLatex(0.21, 0.80, "#sqrt{{s}} = 13 TeV, {:.1f} fb^{{-1}}".format(lumi * args.lumi_scale / 1000.0))
+text.DrawLatex(0.21, 0.80, "#sqrt{{s}} = 13 TeV, {:.2f} fb^{{-1}}".format(lumi * args.lumi_scale / 1000.0))
 
 # Save the plot
-c.SaveAs("WBosonAnalysis.pdf")
+c.SaveAs("df105_WBosonAnalysis.png")
+print("Save figure to df105_WBosonAnalysis.png")
```

Comment:
ROOT will also print that the figure was saved. Sure we need this?

Reply:
It does not for me! But I don't understand why :thinking:

Reply:
@stwunsch It usually gets printed on the error stream. No idea if that's also happening in python.